vsdownload.py

```diff
@@ -626,7 +639,25 @@ def moveVCSDK(unpack, dest):
 
     socket.setdefaulttimeout(15)
 
-    packages = getPackages(getManifest(args))
+    if args.host_arch is None:
+        args.host_arch = platform.machine().lower()
+        if platform.system() == 'Darwin':
+            args.host_arch = 'x64'
```

Comment:
The special treatment of Darwin here probably requires spelling out the reasons, so that we potentially can revisit it in the future.

I presume the reason is that native arm64 Wine on aarch64 doesn't really work on macOS yet, so we prefer x86_64 Wine in Rosetta over broken Wine on aarch64?

Reply:
The reason is simple: there is no prebuilt arm64 Wine on macOS. The package maintainer states (https://github.com/Gcenx/macOS_Wine_builds/issues/74#issuecomment-1858738680):
> There no point build wine for aarch64.

Reply:
> The reason is simple: there is no prebuilt arm64 Wine on macOS.

Yep. (I experimented with building wine for arm64 macOS when this was in developer preview - I am familiar with the issues and challenges around it.)

> The package maintainer states ([Gcenx/macOS_Wine_builds#74 (comment)](https://github.com/Gcenx/macOS_Wine_builds/issues/74#issuecomment-1858738680)):
> 
> > There no point build wine for aarch64.

As the rest of the thread shows - that’s from the point of view if you want to run x86 windows binaries. For running aarch64 windows binaries, it would be entirely reasonable to do.

But the issue indeed is that it’s not available at the moment; mention this in a comment.

Reply:
Done.